### DISCUSSION TITLE
Can I disable a delivery channel for an individual broadcast/notification?

### DISCUSSION THREAD

**Discussion by nick-evocalize** [magicbell/community#176] [Questions] [answered]
We sometimes do not want to send an email for a notification even if it is enabled in user preferences. Specifically, our use-case is that we are a white-labeled application, and email will not be setup for all of our tenants. So if email is not setup for one of them, then we don't want the notification to send out an email. Is there a way we can do this? I don't see an option for that in your API for creating notifications or broadcasts.

**Comment by unamashana** [comment, marked as answer]
Hey @nick-evocalize will these user records in MagicBell contain an email? From your question, I am assuming they would, in which case, what you are looking for isn't an out of the box feature.

Have you tried the [Ping email provider](https://www.magicbell.com/docs/channels/email/ping)? It sends you a webhook when it's time to deliver email, and you can then decide to not act on it for some of your tenants.

**Reply by nick-evocalize** [reply]
Hey @unamashana, thanks for the quick response! That's too bad it isn't an out-of-the-box feature. We are using SES, not ping. Switching email providers would be a pretty heavy lift, especially when you take into account the impact on email reputation (if keeping the same ip addresses isn't an option), so it likely will not be a viable option for us. Is it possible to turn this into a feature request?

**Reply by nick-evocalize** [reply]
Oh, and yes, the the user records will contain an email.

**Reply by unamashana** [reply]
Hey @nick-evocalize you don't need to switch from SES to Ping. Ping acts as a middleman to let you trigger SES on your server, selectively for the tenants you want to. So in essence, it moves the triggering of SES from MagicBell, back to you, but only after running through the entire smart delivery workflow.

We are happy to take this as a feature request. How do you define tenants? What other properties are tenant based? If you can give us a test login to the app, I'd love to try to understand better.

**Reply by nick-evocalize** [reply]
@unamashana I think we've figured out a pretty simple workaround. If we don't want to send an email, we can provide a bogus value in the [provider overrides](https://www.magicbell.com/docs/channels/email/amazon-ses#provider-overrides) section so that SES fails when it tries to send an email. EX we could tell it to use a configuration set that doesn't actually exist.

**Reply by unamashana** [reply]
That's a very interesting workaround. On that note, may be you can force it into sandbox mode? That way you can debug but it's not sent out? Either way, thanks for sharing the very clever hack!
